appveyor.yml

```diff
@@ -0,0 +1,28 @@
+version: '{build}'
+
+environment:
+  matrix:
+    - RUBY_VERSION: 25
+
+install:
+  - ps: . { iwr -useb https://raw.githubusercontent.com/metanorma/metanorma-build-scripts/master/appveyor.ps1 } | iex
+  - refreshenv
+
+build_script:
+  - set PATH=C:\Ruby%RUBY_VERSION%\bin;%PATH%
+  - gem install metanorma-cli
```

Comment:
Do we really want to have this dependency here? Please see my comment here: https://github.com/metanorma/mn-templates-iso/issues/1#issuecomment-487513718 cc: @ronaldtse

Reply:
Impossible to execute metanorma CLI before installing it first)

Reply:
Yap, initially I was considering not to be dependent on the metanorma cli, since `metanorma new` does not do any validation, but as @ronaldtse mentioned in the other thread here https://github.com/metanorma/mn-templates-iso/issues/1#issuecomment-487599701, we want to use the `metanorma compile`, so yes we do need the cli, please ignore this comment!